DSCResources/MSFT_xWebsite/MSFT_xWebsite.psm1

```diff
@@ -323,24 +332,26 @@ function Set-TargetResource
                 $ErrorMessage += $_.Exception.Message
                 $Exception = New-Object -TypeName System.InvalidOperationException -ArgumentList $ErrorMessage
                 $ErrorRecord = New-Object -TypeName System.Management.Automation.ErrorRecord -ArgumentList $Exception, $ErrorId, $ErrorCategory, $null
+
                 $PSCmdlet.ThrowTerminatingError($ErrorRecord)
             }
         }
     }
-    else #Ensure is set to "Absent" so remove website
+    else # Remove website if Ensure is set to 'Absent'
     {
         try
         {
             $Website = Get-Website | Where-Object -FilterScript {$_.Name -eq $Name}
```

Comment:
We do this twice in the highest if/else statement. Would it make sense to just move this above the if/else call and do this once?